jobrunner/manage_jobs.py

```diff
@@ -134,7 +134,9 @@ def create_and_populate_volume(job):
     if config.LOCAL_RUN_MODE and not job.commit:
```

Comment:
I think this now needs to be:
```py
if config.LOCAL_RUN_MODE and not job.commit and not job.action_commit
```
Otherwise running re-usable actions locally won't work.